packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,511 @@
+        const isOutsideMonth = calendarDate.month !== this.currentDate.month;
+
+        const isSelected = Boolean(
+            this.selectedDate &&
+                isSameDay(this.toCalendarDate(this.selectedDate), calendarDate)
+        );
+
+        const isToday = isSameDay(calendarDate, this.today);
+
+        const isDisabled = Boolean(
+            this.disabled ||
+                (this.minDate && calendarDate.compare(this.minDate) < 0) ||
+                (this.maxDate && calendarDate.compare(this.maxDate) > 0)
+        );
+
+        const dayClasses = {
+            'is-outsideMonth': isOutsideMonth,
+            'is-selected': isSelected,
+            'is-today': isToday,
+            'is-disabled': isDisabled,
+        };
+
+        // TODO: The title must include "Today," and " selected" translated to the current language
+        const currentDayTitle = this.formatDate(calendarDate, {
+            weekday: 'long',
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+        });
+
+        return html`
+            <td
+                role="gridcell"
+                class="spectrum-Calendar-tableCell"
+                title=${currentDayTitle}
+                tabindex=${ifDefined(!isOutsideMonth ? '-1' : undefined)}
+                aria-disabled=${isOutsideMonth || this.disabled}
+                aria-selected=${isSelected}
+            >
+                <span
+                    role="presentation"
+                    class="spectrum-Calendar-date ${classMap(dayClasses)}"
+                    @click=${() => this.handleDayClick(calendarDate)}
+                >
+                    ${this.formatNumber(calendarDate.day)}
+                </span>
+            </td>
+        `;
+    }
+
+    public handlePreviousMonth(): void {
+        this.currentDate = startOfMonth(this.currentDate).subtract({
+            months: 1,
+        });
+    }
+
+    public handleNextMonth(): void {
+        this.currentDate = startOfMonth(this.currentDate).add({ months: 1 });
+    }
+
+    public handleDayClick(calendarDate: CalendarDate): void {
+        this.selectedDate = calendarDate.toDate(this.timeZone);
+        this.setCurrentCalendarDate();
+
+        this.dispatchEvent(
+            new CustomEvent('change', {
+                bubbles: true,
+                composed: true,
+                detail: this.selectedDate,
+            })
+        );
+    }
+
+    /**
+     * Defines the array with the indexes (starting at zero) of the weeks of the current month
+     */
+    private setWeeksInCurrentMonth(): void {
+        const numberOfWeeks = getWeeksInMonth(this.currentDate, this.locale);
+
+        this.weeksInCurrentMonth = [...new Array(numberOfWeeks).keys()];
+    }
+
+    /**
+     * Defines the array with data for the days of the week, starting on the first day of the week according to the
+     * defined location (Sunday, Monday, etc.)
+     */
+    private setWeekdays(): void {
```

Comment:
This happens once a render, it might be clearer to make `this.weekdays` a getter that returns the work here, rather than needing to manually call `setWeekdays` each render pass. If there were an alternative path to being able to cache this so it doesn't need to be calculated so often, that would be cool, too.